components/kyma-environment-broker/internal/process/input/builder.go

```diff
@@ -256,13 +256,20 @@ func (f *InputBuilderFactory) CreateUpgradeInput(pp internal.ProvisioningParamet
 		return nil, errors.Wrap(err, "while initializing UpgradeRuntimeInput")
 	}
 
+	kymaInput, err := f.initProvisionRuntimeInput(provider, version)
+	if err != nil {
+		return nil, errors.Wrap(err, "while initializing RuntimeInput")
+	}
+
+
```

Comment:
nit: you may want to run `go fmt components/kyma-environment-broker/internal/process/input/builder.go`, it would get rid of the double empty line